lib/AST/Decl.cpp

```diff
@@ -44,7 +44,7 @@
 #include "swift/AST/TypeCheckRequests.h"
 #include "swift/AST/TypeLoc.h"
 #include "swift/AST/SwiftNameTranslation.h"
-#include "swift/Parse/Lexer.h"
+#include "swift/Parse/Lexer.h" // FIXME: Bad dependency
```

Comment:
What's up here?

Reply:
Just something I noticed while working on the PR, as I understand it, libAST shouldn't depend on libParse.

Reply:
Ah, I read the diff wrong. I thought you had *introduced* this file.